laravel/resources/views/partials/event/slot.blade.php

```diff
@@ -42,7 +42,14 @@
 
 if($start_date->lt(\Carbon\Carbon::now()))
 {
-    $href = "";
+    if (Auth::user()->hasRole('department-lead')||Auth::user()->hasRole('admin')) 
+    {
+
```

Comment:
This is messy. Don't use a blank if statement and an else, just use a negative condition.

Reply:
Because that's not how logic works.